src/components/kudos/GiveKudos.jsx

```diff
@@ -115,16 +119,19 @@ export function GiveKudos({ userData, markStepComplete }) {
 
   useEffect(() => {
     if (givenKudosData) {
-      setKudosGiven(givenKudosData)
+      const dividedKudosData = divideBy10e18(givenKudosData)
+      setKudosGiven(dividedKudosData)
```

Comment:
thoughts on onelining this?
`setKudosGiven(divideBy10e18(givenKudosData))`?

Reply:
I like one liners but think nested functions are ugly. We can go nested one liner here.